docs/index.md

```diff
@@ -928,13 +928,13 @@ To ensure your tests aren't leaving messes around, here are some ideas to get st
 
 ### `--pass-on-failing-test-suite`
 
-> _New in v10.7.0_
+> _Since v10.7.0_
 
 If set to `true`, Mocha returns exit code `0` even if there are failed tests during run.
 
 ### `--fail-zero`
 
-> _New in v9.1.0_
+> _Since in v9.1.0_
```

Comment:
[Typo] Here and elsewhere:

```suggestion
> _Since v9.1.0_
```

Reply:
Thanks for approving my first PR , I'll fix the typos in another PR.